core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-contentlet-tools/ema-contentlet-tools.component.ts

```diff
@@ -67,6 +77,37 @@ export class EmaContentletToolsComponent {
         }
     ];
 
+    ngOnDestroy(): void {
+        // Remove all the appended elements
+        document.querySelectorAll('[data-dot-drag-item]').forEach((el) => {
```

Comment:
Everytime you find yourself doing `querySelector` in an Angular component, think again.

Reply:
Removed